docs/upgrading-fcos.md

```diff
@@ -13,14 +13,125 @@ Occasionally, changes are made to Fedora CoreOS Butane configs (those that speci
 1. TOC
 {:toc}
 
-{% comment %}
-
 ## From Version 1.4.0 to Version 1.5.0
 
 There are no breaking changes between versions 1.4.0 and 1.5.0 of the `fcos` configuration specification. Any valid 1.4.0 configuration can be updated to a 1.5.0 configuration by changing the version string in the config.
 
 The following is a list of notable new features.
 
+## GRUB password
+
+A new `grub` section was added with an array of users can be set with `name`(s) and `password_hash`(s). Any user that does not have the given usernames and passwords will not be able to use the GRUB command line, modify kernel command-line arguments, or boot non-default OSTree deployments.
+
+<!-- butane-config -->
+```yaml
+variant: fcos
+version: 1.5.0
+grub:
+  users:
+    - name: root
+      password_hash: grub.pbkdf2.sha512.10000.874A958E5264...
+```
+
+### Offline Tang provisioning
+
+The `tang` sections in `storage.luks` and `boot_device.luks` gained a new advertisement field. If specified, Ignition will use it to provision the Tang server binding rather than fetching the advertisement from the server at runtime. This allows the server to be unavailable at provisioning time. The advertisement can be obtained from the server with `curl http://tang.example.com/adv`.
```

Comment:
```suggestion
The `tang` sections in `storage.luks` and `boot_device.luks` gained a new `advertisement` field. If specified, Ignition will use it to provision the Tang server binding rather than fetching the advertisement from the server at runtime. This allows the server to be unavailable at provisioning time. The advertisement can be obtained from the server with `curl http://tang.example.com/adv`.
```